### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by xXEmilioXx**

### CODE REVIEW

**Comment by Frikkha** [comment]
Is it correct to write that we use jest for testing the web-application? 
Didn't the web-team say they havn't written any tests?

**Comment by xXEmilioXx** [comment]
From what I understood they tried to write tests with jest but they won't achieve the 80% coverage. I don't know if I understood wrong

**Comment by Frikkha** [comment]
Yes they said they tried, but as far as i can see on github there is only one test app.test.js which is hardly a test. Maybe someone has some local tests that is not comitted yet

File: README.md [hunk 1]
```diff
@@ -1,2 +1,58 @@
...
+
+This course aims at assessing the students' ability to design and implement a complex software system by working togheter from different places in the world (some students are from POLIMI, which is in Milan (Italy), some others are from FER, which is in Zagreb (Croatia)). The requirements were given by a customer, which was a Senior developer in Deloitte Digital, so another assessed ability was being able to constantly contact him and satisfy his requests. The working framework used is SCRUM, so the students' ability to organize meetings and respect the SCRUM rules was also assesed. 
+
+This repository refers to the 2019-2020 edition of the course, and the project assigned is "TheSpoon": an application that lets restaurant owners to register with their restaurant data and its menus, while customers can search specific food they want to eat and rate the restaurants. Another type of user, the consultant, can access the statistics of the application and monetize them by helping the restaurant owners improving their activity. More precisily, the project included:
+- A web application used by the restaurant owners and the consultants
+- A mobile app used bu the customers
```
Comment (cathrbak) [inline README.md]: typo, by, not bo

File: README.md [hunk 2]
```diff
@@ -1,2 +1,58 @@
...
+
+The folder of the mobile app is "mobileApp". Inside that there is another README which explains how to run it.
+
+The project has been developed with the following technologies:
+- Server hosting: Heroku
+- Backend: NodeJS
+- API reference: Swagger
+- Database: Postgres
+- Frontend web: React
+- Frontend mobile: React native
```
Comment (cathrbak) [inline README.md]: change to: "React native with expo"

File: README.md (hunk 1 above)
Comment (xXEmilioXx) [reply README.md]: fixed

File: README.md [hunk 3]
```diff
@@ -1,2 +1,58 @@
...
+- Testing tool used to test backend, web and mobile: Jest
```
Comment (idaame) [inline README.md]: Should we mention that the mobileApp uses Enzyme in addition to Jest?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,58 @@
...
+
+The folder of the mobile app is "mobileApp". Inside that there is another README which explains how to run it.
+
+The project has been developed with the following technologies:
+- Server hosting: Heroku
+- Backend: NodeJS
+- API reference: Swagger
+- Database: Postgres
+- Frontend web: React
+- Frontend mobile: React native
```
Comment (idaame) [inline README.md]: Change to "React Native with Expo"

Comment (xXEmilioXx) [reply README.md]: I already did it :)

Comment (xXEmilioXx) [reply README.md]: I actualy wrote "React native with expo", I will also put the capital letters

File: README.md (hunk 3 above)
Comment (xXEmilioXx) [reply README.md]: we should, now I fix it

**Comment by xXEmilioXx** [comment]
> Yes they said they tried, but as far as i can see on github there is only one test app.test.js which is hardly a test. Maybe someone has some local tests that is not comitted yet

I don't know, for the moment I write that they use it, we can always change the readme in case we discover they completely left using jest

File: README.md (hunk 3 above)
Comment (xXEmilioXx) [reply README.md]: Is it ok now?

File: README.md [hunk 5]
```diff
@@ -1,2 +1,60 @@
...
+- Frontend mobile: React Native with Expo
+- Testing the mobile app: Jest, Enzyme
+- Continuous integration: Travis CI
+- Textual communication: Slack
+- Meetings: Zoom
+
+## Group
+| First name | Last Name | University | Subteam         |
+| ---------- | --------- | ---------- | --------------- |
+| Ida Merete | Enholm    | POLIMI     | Mobile app      |
```
Comment (idaame) [inline README.md]: Change to "Mobile app + UI Design" ?

Comment (xXEmilioXx) [reply README.md]: ok of course

File: README.md (hunk 3 above)
Comment (idaame) [reply README.md]: Yes :D

File: README.md (hunk 5 above)
Comment (xXEmilioXx) [reply README.md]: done

Comment (xXEmilioXx) [reply README.md]: maybe I should also add the product owner + scrum master roles?

**Pull request merged**